config/src/config/consensus_config.rs

```diff
@@ -59,6 +74,38 @@ impl Default for ConsensusConfig {
             quorum_store_pull_timeout_ms: 1000,
             quorum_store_poll_count: 10,
             intra_consensus_channel_buffer_size: 10,
+
+            window_for_chain_health: 100,
+            chain_health_backoff: vec![
+                ChainHealthBackoffValues {
+                    backoff_if_below_participating_voting_power_percentage: 80,
+                    proposer_create_proposal_backoff_ms: 100,
+                    round_timeout_backoff_ms: 200,
+                    max_sending_block_txns_override: 2000,
+                    max_sending_block_bytes_override: 500 * 1024,
+                },
+                ChainHealthBackoffValues {
+                    backoff_if_below_participating_voting_power_percentage: 75,
+                    proposer_create_proposal_backoff_ms: 300,
+                    round_timeout_backoff_ms: 600,
+                    max_sending_block_txns_override: 1000,
+                    max_sending_block_bytes_override: 400 * 1024,
+                },
+                ChainHealthBackoffValues {
+                    backoff_if_below_participating_voting_power_percentage: 72,
+                    proposer_create_proposal_backoff_ms: 500,
+                    round_timeout_backoff_ms: 1000,
+                    max_sending_block_txns_override: 400,
```

Comment:
round_timeout_backoff_ms alone doesn't help enough unless set very aggressively (especially if we increase default block size to safely increase TPS after this lands).

That is the case because longer round_timeout_backoff_ms makes it such that we have more full blocks. it only kicks in after blocks are full and we increase it further. 

Also having just round_timeout_backoff_ms might increase commit latency, as it will generate larger blocks. 

If we don't want to increase block size to increase max TPS together with this change, then I would be more comfortable with having round_timeout_backoff_ms only. But that gives us also half the benefit of this change - we gracefully backoff when terrible, but not increase TPS on happy path.

Reply:
After quorum store, there is still a block size, and it still controls the throughput given to the execution.
Basically TPS is always: block size * consensus latency, where consensus latency is basically time between consecutive blocks. that is both before and after quorum store.
so I think this back-pressure applies with no changes both now and in the future